src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
@cwe1ss - Asp.Net Core supports .NET451 minimally (which maps to `netstandard1.2`). However `netstandard` exposes a subset of APIs that NET451 supports. In cases where we are unable to target a low enough netstandard to support NET451, we cross compile it.

I wrote some notes when I was converting `dnxcore50` -> `dotnet5.x` here (https://gist.github.com/pranavkm/1f2786f91a2e065ca0bb) which lists some of these APIs that caused us to cross compile. In this case specific case, `Assembly` exists in NETStandard1.3`, but `Assembly.Location` is only available in `NETStandard1.5`